app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationFilters.tsx

```diff
@@ -141,7 +139,7 @@ export const ConversationFilters = ({
       <EventFilter selectedEvents={filterValues.events} onChange={(events) => onUpdateFilter({ events })} />
       <PromptFilter isPrompt={filterValues.isPrompt} onChange={(isPrompt) => onUpdateFilter({ isPrompt })} />
       {activeFilterCount > 0 && (
-        <Button variant="ghost" onClick={onClearFilters}>
+        <Button id="clear-filters-button" variant="ghost" onClick={onClearFilters}>
```

Comment:
```suggestion
        <Button data-testid="clear-filters-button" variant="ghost" onClick={onClearFilters}>
```

(so it's clear this is for testing, [it's a convention in Playwright](https://playwright.dev/docs/locators#locate-by-test-id))